ONSdigital/eq-survey-runner
# EQ-387: Add ability to load schemas from s3

### Changes

This PR enables the survey runner to load schemas from s3. A new environment variable is
added called EQ_SCHEMA_BUCKET which if set, tries to load schemas from an s3 bucket, using
IAM permissions when deployed to Elasticbeanstalk or picks up standard AWS boto credential
set.
### How to test
1. Create a new s3 bucket and upload a json schema to it, ensuring that the schema is not named the same as any existing schemas.
2. Set the environment variable `EQ_SCHEMA_BUCKET` to the name of the bucket.
3. Run the application and visit the dev page to view the list of schemas.
4. Complete a survey and verify it submits ok.
### Who can test

Anyone but @dhilton
